src/main/java/seedu/weme/logic/parser/MemeExportCommandParser.java

```diff
@@ -0,0 +1,43 @@
+package seedu.weme.logic.parser;
+
+import static seedu.weme.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.weme.logic.parser.CliSyntax.PREFIX_FILEPATH;
+
+import java.util.stream.Stream;
+
+import seedu.weme.logic.commands.ExportCommand;
+import seedu.weme.logic.parser.exceptions.ParseException;
+import seedu.weme.model.DirectoryPath;
+
+/**
+ * Parses input arguments and creates a new MemeExportCommand object
+ */
+public class MemeExportCommandParser implements Parser<ExportCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the MemeExportCommand
+     * and returns an MemeExportCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public ExportCommand parse(String args) throws ParseException {
+        ArgumentMultimap argMultimap =
+                ArgumentTokenizer.tokenize(args, PREFIX_FILEPATH);
+
+        if (!arePrefixesPresent(argMultimap, PREFIX_FILEPATH)
```

Comment:
Only one prefix is needed here. This can be checked directly without using `arePrefixesPresent`. Even if `arePrefixesPresent` is to be used, it should be pulled out into `ParserUtil` and imported instead of copy/pasted here.